modules/administration/pages/confidential-computing.adoc

```diff
@@ -0,0 +1,112 @@
+[[confidential_computing]]
+= Confidential Computing 
+
+{CoCo} is a technology which allows protection of data in use by using hardware-based Trusted Execution Environment (TEE), the type of environments that provide increased level of security for data integrity, data confidentiality, and code integrity.
+
+
+
+== {CoCo} with {productname}
+
+The trustworthiness of the TTE is checked with the attestation process.
+{productname} can be used as an attestation server for the systems registered to it.
+It generates a report page for the systems which run in this mode.
+These systems need to be attested and checked on regular base.
+The history of the past checks is also stored and available per request.
+
+{CoCo} Attestation depends on the used hardware and environment where the attested systems are running on.
+
+
+
+== Requirements
+
+{CoCo} can be set up in an environment with the following characteristics:
+
+* Attested system (virtual machine) is *SLES15 SP6* and bootstrapped to {productname}
+* Hardware must have *AMD EPYC Milan CPU* or *AMD EPYC Genoa CPU*
+
+//// 
+This was from the testing guidelines, perhaps it is not needed in the documentation?
+* [literal]``kernel-default`` must be installed instead of [literal]``kernel-default-base``.
+  If necessary, run:
++
+----
+zypper install -- -kernel-default-base +kernel-default
+----
+////
+
+
+=== Limitations
+
+{CoCo}  Attestation is currently implemented only as a technological preview. 
+//OM: (Do we state this explicitly in the documentation?)
```

Comment:
We should add a link to the setup procedure for the HostOS as soon as such a documentation exists.
But maybe you can write already a sentence like:

"How to setup and configure CoCo on your Host, please have a look at the OS Vendor documentation"